docs/security.rst

```diff
@@ -72,6 +72,7 @@ New releases for OpenSSL updates
 --------------------------------
 
 As of versions 0.5, 1.0.1, and 2.0.0, ``cryptography`` statically links OpenSSL
+in :ref`binary distrubtions <installation:Static Wheels>`
```

Comment:
I attempted to link to the https://cryptography.io/en/latest/installation/#static-wheels section of the docs, since it's relevant, but I just realized the way I linked to it requires the `sphinx.ext.autosectionlabel` extension which it looks like we don't have installed. Would you like a PR to add that extension before merging this?

Reply:
Noticed I had a type too 🤦  - fixed and removed the link in the latest revision.